index.html

```diff
@@ -1,140 +1,170 @@
-							<a class="button featurebutton" href="#contribute">Contribute</a>
-						</div>
-					</div>
-				</article>
-			</section>
-		</header>
-		<section class="details">
-			<article>
-				<h2>Tools &amp; Features</h2>
-				<div class="content">
-					<div class="vcentercontainer detailstext">
-						<div>
-							<p>Git for Windows focuses on offering a lightweight, native set of tools that bring the full feature set of the <a href="https://git-scm.com" target="_blank">Git SCM</a> to Windows while providing appropriate user interfaces for experienced Git users and novices alike.</p>
-							<h3 id="bash" >Git BASH</h3>
-							<p>Git for Windows provides a BASH emulation used to run Git from the command line.  *NIX users should feel right at home, as the BASH emulation behaves just like the "git" command in LINUX and UNIX environments.</p>
-							<h3 id="git-gui" >Git GUI</h3>
-							<p>As Windows users commonly expect graphical user interfaces, Git for Windows also provides the Git GUI, a powerful alternative to Git BASH, offering a graphical version of just about every Git command line function, as well as comprehensive visual diff tools.</p>
-							<h3 id="explorer" >Shell Integration</h3>
-							<p>Simply right-click on a folder in Windows Explorer to access the BASH or GUI.</p>
-						</div>
-					</div>
-					<div class="vcentercontainer detailsimg">
-						<div class="verticalgallery">
-							<a class="fancybox-thumb" data-fancybox-group="gallery" href="img/gw1.png" title="Git BASH">
-								<img src="img/gw1web_thumb.png" alt="Git BASH">
-							</a>
-							<a class="fancybox-thumb" data-fancybox-group="gallery" href="img/gw2.png" title="Git GUI">
-								<img src="img/gw2web_thumb.png" alt="Git GUI">
-							</a>
-							<a class="fancybox-thumb" data-fancybox-group="gallery" href="img/gw3.png" title="Gitk: Exploring Commits">
-								<img src="img/gw3web_thumb.png" alt="Gitk: Exploring Commits">
+						</p>
+						<p>
+							<a name="download" /><a class="btn btn-primary shadow mr-2"
+								href="https://github.com/git-for-windows/git/releases/latest"
+								target="_blank">Download</a>
+							<a class="btn btn-outline-primary shadow mr-2" href="#contribute">Contribute</a>
+						</p>
+						<p class="h2" id="version">
+							<a href="https://github.com/git-for-windows/git/releases/tag/v2.34.0.windows.1" title="Version 2.34.0 was published on Mon, 15 Nov 2021 19:52:38 UTC">
+								Version 2.34.0
 							</a>
-						</div>
+						</p>
 					</div>
 				</div>
-				<div class="stud"></div>
-				<h2 id="contribute">Contribute</h2>
-				<div class="content contributecontent">
-					<div class="vcentercontainer detailsimg">
-						<div>
-							<img src="img/details_features.png" alt="BASH Drawing">
-							<img src="img/details_contribute.png" alt="Git for Windows Logo">
-						</div>
+			</div>
+		</div>
+	</section>
+	<section>
+		<div class="container">
+			<h2>Tools &amp; Features</h2>
```

Comment:
I like that these now lay out side by side rather than stacked. However, the images aren't the same height, and so there's a ["bunting"](https://www.google.com/search?q=hang+bunting) effect that isn't pleasing.

<img width="1131" alt="patriotic screenshots" src="https://user-images.githubusercontent.com/714283/142431372-8b63db0a-245f-4bc5-b0c2-9e97e9193f60.png">

Reply:
For the record, I just spent an incredible 1h15m on re-generating the screenshots and on opening #33.